test/data/brave_ads/search_result_ad_sample.html

```diff
@@ -0,0 +1,62 @@
+<html>
+<head>
+  <title>Brave Search</title>
+  <script type="application/ld+json">
+    {
+      "@context": {
+         "@vocab": "https://schema.org",
+         "creatives": null,
+         "data-placement-id": null,
+         "data-creative-instance-id": null,
+         "data-creative-set-id": null,
+         "data-campaign-id": null,
+         "data-advertiser-id": null,
+         "data-landing-page": null,
+         "data-headline-text": null,
+         "data-description": null,
+         "data-rewards-value": null,
+         "data-conversion-type-value": null,
+         "data-conversion-url-pattern-value": null,
+         "data-conversion-advertiser-public-key-value": null,
+         "data-conversion-observation-window-value": null
+     },
+      "@type": "Product",
+      "creatives": [
```

Comment:
This is test data

Reply:
Yeah that made sense to me, but since the test data had terms that get dropped in the document when processed by a JSON-LD processor I wasn't sure what the impact of it was going to be on the code later on. One of the reasons I was trying to get a better understanding of this here was to see if there was any concern for an attacker to do semantic overwriting of real data if it's being processed or if there's concerns of the context being used in a way to conduct CSRF style attacks when trying to resolve remote contexts.

The test data here was making it a bit murky for me to understand how this JSON-LD metadata would get used and if the semantics from JSON-LD mattered or if we were actually putting this through a JSON-LD function like expand. I wasn't overly concerned about the specifics of this exact test data.

Reply:
Seems like I used wrong @context for test json data. I think in final solution we will use our own json-ld scheme. So for now I will delete it in testing data. 
BTW, json-ld metadata parser in chromium doesn't do anything with @context field and just ignores it. So removing it in test data is okay.

Reply:
Ahh very interesting so they're not actually processing the JSON-LD and instead just rely on the JSON parsing of things. That seems very safe then which should avoid any of my concerns since from the sounds of it they would not actually relying on the semantics in any way nor resolving remote contexts.